### ISSUE TITLE
Watir website is offline.

### ISSUE THREAD

**Issue by nfstern02** [watir/watir#974]
## Steps to reproduce - try going to www.watir.com

**Comment by jkotests** [comment]
@titusfortner - Do you know who used to own the domain and if they will be renewing it? 

For now, I've removed the custom domain associated with the watir.github.io pages. This allows the website to be viewable at https://watir.github.io/.

**Comment by titusfortner** [comment]
Yeah, I reached out to the person who has been generously renewing it for 20 years now.

Thanks.

**Comment by nfstern02** [comment]
Thanks, I can reach via watir.github.io

**Comment by titusfortner** [comment]
Huh, ok, that was easier, if more expensive, than I thought it would be.

I can't get a hold of the owner, but his domain registrar let me pay for the renewal (and the expensive recovery fee). But it'll be ready later today, and safe for 2 years. Hopefully I can get in touch with him so it can be properly transferred.

**Comment by titusfortner** [comment]
@jkotests I can't figure out how to put back the custom domain. It might be related to https settings. Do you have availability to look at it?

Also, she said it might take several hours for the DNS to repropagate

**Comment by jkotests** [comment]
@titusfortner - When I tried adding the custom domain the DNS check said, "Both watir.com and its alternate name are improperly configured. Domain does not resolve to the GitHub Pages server. For more information, see [documentation](https://docs.github.com/articles/setting-up-a-custom-domain-with-github-pages/) (NotServedByPagesError)."

Following through the documentation (https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site/managing-a-custom-domain-for-your-github-pages-site#configuring-an-apex-domain), it looks like we need to:
* Steps 1-4: Configure the repository (Done)
* Step 5: Configure at least one ALIAS, ANAME, or A record with your DNS provider - Step 5

Given the error message, it sounds like we are missing configuration on the DNS provider side. Looks like the steps to update Rebel are - https://support.rebel.com/hc/en-us/articles/227817687-What-are-Address-A-Records-and-how-do-I-edit-add-them . Can you check that the 4 GitHub Pages IPs are added?

**Comment by titusfortner** [comment]
Well, that might not be possible... Can you message me on slack?
